orchestrator/commands/command.py

```diff
@@ -0,0 +1,29 @@
+from enum import IntEnum
+
+
+class CommandType(IntEnum):
+    CHECK_STATUS_ON_LSF = 0
+    SUBMIT = 1
+    ABORT = 2
+    SUSPEND = 3
+    RESUME = 4
```

Comment:
I'm a little afraid of this. The `ABORT/SUSPEND/RESUME` makes sense, but we had just split `check_status` and `submit` due to the SUBMIT clobbering everything else in the queue and wanting to enforce check_status can only run on a single worker. I don't see these implemented but curious what you plan on doing with it.

Reply:
I used `memcache_task_lock` to ensure that only one worker can do one Command on one Job. So we can't have the situation that we are checking the status, and in the middle of that abort command is executed by other worker, and we don't transition properly. Also this way we are executing check_status_of_the_jobs in parallel, which looked much faster when I tested it on Staging. Each command is basically this task https://github.com/mskcc/ridgeback/pull/199/files#diff-8e999f74b408fe7717f1e597acafb3105363ad618bc161bb935135794b301ed6R109-R134, and this https://github.com/mskcc/ridgeback/pull/199/files#diff-8e999f74b408fe7717f1e597acafb3105363ad618bc161bb935135794b301ed6R113 protect us that two workers execute same job. Thing I would need to add is the LSF status cache in order to reduce the amount of calls on LSF, since now multiple workers are asking LSF.